content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,243 @@
+export KO_DOCKER_REPO=gcr.io/$PROJECT_ID
+ko apply -f falcoeventlistener.yaml
+```
+
+> If you are using private container registry, don't forget to create a registry secret to be able to pull and push images. You can follow the following guide to achieve this:                                                       
+https://colinwilson.uk/2020/07/09/using-google-container-registry-with-kubernetes/
+
+If everything works well, we should see the similar output below:
+
+```shell
+$ kubectl get pods --namespace falco
+NAME                   READY   STATUS    RESTARTS   AGE
+falco-event-listener   1/1     Running   0          59s
+```
+
+The last thing we have to do is installing `Falco`, `Falcosidekick` and configuring `Falcodekick` to forward events to our project.
+
+### Installing Falco and Falcosidekick
+
+There is a up-to-date and detail guide available to install `Falco` and `Falcosidekick`, you can follow this [link](https://github.com/falcosecurity/charts/blob/master/falcosidekick/README.md#installing-the-chart) to learn more about installing them.
+
+Here is that we are going to enable `CloudEvents` support of Falcosidekick.
+
+```shell
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+helm repo update
+helm upgrade --install falco falcosecurity/falco \
+  --namespace falco \
+  --set falcosidekick.enabled=true \
+  --set falcosidekick.webui.enabled=true \
+  --set falcosidekick.config.cloudevents.address=http://falco-event-listener
+```
+
+Verify if everything is working before move on to the next step.
+
+```shell
+$ kubectl get pods --namespace falco
+NAME                                      READY   STATUS    RESTARTS   AGE
+falco-event-listener                      1/1     Running   0          5m19s
+falco-falcosidekick-5854669c76-ddvrv      1/1     Running   0          2m16s
+falco-falcosidekick-5854669c76-rdlqn      1/1     Running   0          2m16s
+falco-falcosidekick-ui-7c5fc8dd54-q4qh9   1/1     Running   0          2m16s
+falco-vkl4f                               1/1     Running   0          2m16s
+```
+
+## Test
+
+To test this, we have to connect a shell within the container.
+
+Let's list the pods that we already have.
+
+```shell
+$ kubectl get pods
+NAME                                 READY   STATUS    RESTARTS   AGE
+alpine-deployment-77789455d6-m7flp   1/1     Running   15         3h6m
+alpine-deployment-77789455d6-v7fkw   1/1     Running   15         3h6m
+podinfo-6df788c7b8-gs5qb             1/1     Running   0          3h28m
+podinfo-6df788c7b8-sfxvd             1/1     Running   0          3h28m
+```
+
+Now, run the following command to connect a shell.
+
+```shell
+$ kubectl exec -ti alpine-deployment-77789455d6-m7flp -- sh -c "uptime"
+19:35:58 up  3:04,  load average: 3.12, 1.91, 1.22
+```
+
+Once you run the command above, Falco will detect that malicious behavior and send it to the Falcosidekick via HTTP.
+
+You should see the similar output below in the `Falco` logs:
```

Comment:
You should see an output in the `Falco` logs as given below:

Reply:
thank you so much for fixing the issues, done.